R/plot_jags_densitydx.R

```diff
@@ -26,26 +26,10 @@
 #' - `y1` = posterior estimate of peak antibody concentration
 #' @param strat Specify [character] string to produce plots of specific
 #' stratification entered in quotes.
-#' @return A [base::list()] of [ggplot2::ggplot()] objects producing density
+#' @return A [list] of [ggplot2::ggplot] objects producing density
 #' plots for all the specified input.
 #' @export
-#' @examples
-#' if (!is.element(runjags::findjags(), c("", NULL))) {
-#'   library(runjags)
-#'   library(ggmcmc)
-#'   library(dplyr)
-#'
-#'   data <- serodynamics::nepal_sees_jags_post
-#'
-#' plot_jags_dens(
-#'     data = data, #A [serodynamics::run_mod()] [list] output.
-#'     iso = "HlyE_IgA", #A [character] string specifying
-#'     #nantigen/antibody of interest.
-#'     param = "alpha",  #A [character] string specifying parameter of
-#'     # interest.
-#'     strat = "typhi")  #A [character] string specifying
-#'     # stratification of interest.
-#'     }
+#' @examples inst/examples/examples-plot_jags_densitydx.R
```

Comment:
```suggestion
#' @example inst/examples/examples-plot_jags_densitydx.R
```
This is a frustrating convention from roxygen2: `@examples` is for inline examples, `@example` (no `s`) is for linking to external example files